modules/ROOT/pages/ogm/installation.adoc

```diff
@@ -287,11 +305,11 @@ In your application directory, you can run this application:
 node index.js
 ----
 
-Which should output:
+You should see the following output:
 
 [source, bash, indent=0]
 ----
-Example app listening at http://localhost:4000
+Example app listening at http://localhost:4000/users
 ----
 
-The REST API should now be ready to accept requests at the URL logged.
+The REST API should now be available at `http://localhost:4000`, with  a single working route `/users`.
```

Comment:
We shouldn't use ` for links, even though I understand this is the URL of the database/API

```suggestion
The REST API should now be available at http://localhost:4000, with a single working route `/users`.
```